e2e/tests/popupInMenu-example.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { Menu, Popup } from '@stardust-ui/react'
+
+export const selectors = {
+  menuId: 'menu',
+  menuItemId: index => `menu-item-${index}`,
+  popupContentClass: Popup.slotClassNames.content,
+  popupContentId: index => `popup-content-${index}`,
+}
+
+const items = [
+  { key: 'editorials', content: 'Editorials' },
+  { key: 'review', content: 'Reviews' },
+  { key: 'events', content: 'Upcoming Events' },
+]
+
+const PopupInMenuExample = () => (
+  <Menu
+    id={selectors.menuId}
+    defaultActiveIndex={0}
+    items={items.map((item, index) => render =>
+      render(item, (MenuItem, props) => renderItem(MenuItem, props, index)),
+    )}
+  />
+)
+
+const renderItem = (MenuItem, props, index) => {
+  return (
+    <Popup
+      trigger={<MenuItem id={selectors.menuItemId(index)} {...props} />}
```

Comment:
```suggestion
      key={props.key}
      trigger={<MenuItem id={selectors.menuItemId(index)} {...props} />}
```

Missing `key`, it's a second case with propTypes error. Can we add a check somehow that there are no any errors in console?

You can take a look on `performBrowserTest` in `build/gulp/tasks/test-projects.ts`

Reply:
yes, will add this check, thanks!